package.json

```diff
@@ -122,6 +122,7 @@
     "body-parser": "^1.15.2",
     "chai": "^4.1.1",
     "chai-as-promised": "^7.1.1",
+    "chalk": "^2.4.2",
```

Comment:
We're [already](https://github.com/department-of-veterans-affairs/vets-website/blob/3e7daca36ae04b7263eb12f0d889c75f305391ed/script/watch.js#L9) using this.